.github/workflows/keep-alive.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     name: Alive
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@1b33e4ef553c59eef0e3450666408021f4b0f456
       - uses: gautamkrishnar/keepalive-workflow@v1
```

Comment:
This should have been added to `keepalive-workflow`. Fixing in PR ( https://github.com/conda-forge/webservices-dispatch-action/pull/24 )